resources/views/news/index-atom.blade.php

```diff
@@ -0,0 +1,47 @@
+{{--
+    Copyright (c) ppy Pty Ltd <contact@ppy.sh>.
+
+    This file is part of osu!web. osu!web is distributed with the hope of
+    attracting more community contributions to the core ecosystem of osu!.
+
+    osu!web is free software: you can redistribute it and/or modify
+    it under the terms of the Affero GNU General Public License version 3
+    as published by the Free Software Foundation.
+
+    osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+    warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+    See the GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+--}}
+{{-- just in case php has shorttag enabled --}}
+{!! '<?xml version="1.0" encoding="UTF-8"?>' !!}
+<feed xml:lang="en-US" xmlns="http://www.w3.org/2005/Atom">
+    <id>tag:osu,2019:news</id>
+
+    <link rel="alternate" type="text/html" href="{{ route('news.index') }}" />
+    <link rel="self" type="application/atom+xml" href="{{ request()->fullUrl() }}" />
+
+    <title>osu!news</title>
+    <icon>{{ config('osu.static') }}/apple-touch-icon.png</icon>
+
+    <updated>{{ json_time(optional($posts->last())->published_at ?? now()) }}</updated>
+
+    @foreach ($posts as $post)
+        <entry>
+            <id>tag:osu,2019:news/{{ $post->getKey() }}</id>
```

Comment:
shouldn't the tags (here and above) be `tag:osu.ppy.sh` or something more specific?